pubspec.yaml

```diff
@@ -22,45 +22,48 @@ dependencies:
     sdk: flutter
 
   auto_route: 7.4.0
-  bloc: 8.1.1
   cupertino_icons: 1.0.5
   dartx: 1.1.0
-  dio: 5.1.1
+  dio: 5.1.2
   equatable: 2.0.5
-  floor: 1.4.1
-  flutter_bloc: 8.1.2
-  flutter_dotenv: 5.0.2
-  flutter_native_splash: 2.2.19
+  floor: 1.4.2
+  flutter_bloc: 8.1.3
+  flutter_dotenv: 5.1.0
+  flutter_native_splash: 2.3.1
```

Comment:
did you run the command again? When you update it you have to run project_setup.sh script